<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wtraDialog</class>
 <widget class="QDialog" name="wtraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New WTRA</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>270</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Emitters to Risk Assess</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Justification</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dtgPTTX"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="justificationBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Department:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Expiry of WTRA</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateTimeEdit" name="dtgExpiry"/>
    </item>
    <item row="2" column="1">
     <widget class="QListWidget" name="emitterList"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Big E</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ShipHaz Activity Line</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>PTTX DTG</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="shActivities"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>shActivities</tabstop>
  <tabstop>emitterList</tabstop>
  <tabstop>justificationBox</tabstop>
  <tabstop>dtgPTTX</tabstop>
  <tabstop>dtgExpiry</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>wtraDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>wtraDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shActivities</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>wtraDialog</receiver>
   <slot>listHazards()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>listHazards()</slot>
 </slots>
</ui>
